pyproject.toml

```diff
@@ -84,7 +84,7 @@ testpaths = [ "tests", ]
 files = ["src", "tests"]
 exclude = "class_soc_calc\\.py$"
 check_untyped_defs = true
-warn_unused_ignores = true
+warn_unused_ignores = false
```

Comment:
According to https://github.com/python/mypy/issues/8823 it is possible to add `ignore[unused-ignore]`, e.g. `# type: ignore[attr-defined,unused-ignore]`. This seem to work well on Linux when I add it to eos.py and test_server.py (both Linux and Windows code paths).